packages/react-checkbox/src/components/Checkbox/useCheckboxStyles.ts

```diff
@@ -4,226 +4,197 @@ import { CheckboxState } from './Checkbox.types';
 
 export const checkboxClassName = 'fui-Checkbox';
 
+// TODO replace these spacing constants with theme values once they're on the theme
+const spacingHorizontalS = '8px';
+const spacingHorizontalM = '12px';
+
+// CSS variables for the indicator colors.
+// These let the colors be picked based on the :hover, etc. states on the root
+const indicatorColor = '--fui-Checkbox-indicator-color';
+const indicatorBorderColor = '--fui-Checkbox-indicator-borderColor';
```

Comment:
FYI none of these styles below actually require css vars to be achieved. We should reduce our css var impact as much as possible since it can add perf consequences. Reducing css var impact was the motivation behind #20041

Reply:
Ok I'll remove the css vars